packages/fxa-settings/src/pages/Signin/en.ftl

```diff
@@ -0,0 +1,27 @@
+## Signin page
+
+# Strings within the <span> elements appear as a subheading.
+signin-password-needed-header = Enter your password <span>for your { -product-firefox-account }</span>
+
+# $serviceLogo - an image of the logo of the service which the user is authenticating for.
+# For languages structured like English, the phrase can read "to continue to"
+signin-subheader-with-logo = Continue to <span>{ $serviceLogo }</span>
+
+# $serviceName - the name of the service which the user authenticating for
+# For languages structured like English, the phrase can read "to continue to { $serviceName }"
+signin-subheader-without-logo-with-servicename = Continue to { $serviceName }
+signin-subheader-without-logo-default = Continue to account settings
+signin-button = Sign in
+signin-header = Sign in
+
+# This message is followed by a bulleted list
+signin-tos-list-intro = By proceeding, you agree to:
+signin-tos-list-pocket = { product-pocket }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>
+signin-tos-list-firefox = { -brand-firefox }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal># <linkExternal> links to the Terms of Service and also to the Privacy Notice
```

Comment:
Q should the trailing comment be up on L18.5 or somethin?
```
# <linkExternal> links to the Terms of Service and also to the Privacy Notice
```

Reply:
If we really wanted to be thorough, it could go before each instance of <linkExternal> so before:
signin-tos-list-pocket
signin-tos-list-firefox
tos-and-privacy

Reply:
I'll add that in!